docs/architecture/microservices/secure-net-microservices-web-applications/azure-key-vault-protects-secrets.md

```diff
@@ -43,7 +43,7 @@ Note that calling `AddAzureKeyVault` requires the application ID that was regist
   [https://docs.microsoft.com/aspnet/core/security/data-protection/configuration/default-settings](/aspnet/core/security/data-protection/configuration/default-settings)
 
 - **Microsoft.Extensions.Configuration** GitHub repository. \
-  <https://github.com/dotnet/extensions/tree/master/src/Configuration>
+  <https://github.com/dotnet/extensions/tree/main/src/Configuration>
```

Comment:
This link appears to be broken on both master and main.

Reply:
The link was removed in https://github.com/dotnet/docs/pull/23241.